src/plays/travel-advisor/styles/dropdown.css

```diff
@@ -0,0 +1,40 @@
+.select {
```

Comment:
Please prefix the class names with your play name as you have done for `travel-advisor-main-Container